### DISCUSSION TITLE
interrupted FDS Simulation

### DISCUSSION THREAD

**Discussion by LordMicidial** [firemodels/fds#12774] [Q&A] [answered]
Dear all,

I was waiting my simulation to finish (900 s). It took 1 week and a half to get to 705 s.

This morning I opened the PC and I've noticed that the CMDfds windows just disappeared! i'm very frustrated. What shall I check to understand what happened? My PC is brand new (bought for simulation purposes)

I'm afraid I might have close the process while the PC screen was off. I really can't understand. The out file can't tell why process stopped.

Any Idea? I really don't want to loose 1 week time again.

**Comment by RIO4FDS** [comment, marked as answer]
Hi,
Do you use Windows as your OS?
Automatic updates (and restart) could be a potential problem.
Regards,

On Thu, Apr 11, 2024 at 5:16 AM LordMicidial ***@***.***> wrote:
>
> Dear all,
>
> I was waiting my simulation to finish (900 s) took 1 week and a half to get to 705 s.
>
> This morning I opened the PC and I've noticed that the CMDfds windows just disappeared! i'm very frustrated. What shall I check to understand what happened? My PC is brand new (bought for simulation purposes)
>
> I'm afraid I might have close the process while the PC screen was off. I really can't understand. The out file can't tell why process stopped.
>
> Any Idea? I really don't want to loose 1 week time again.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>



-- 
___________________________________
Julio Cesar Goncalves da Silva, D.Sc.
www.linkedin.com/in/juliosilvafire
sites.google.com/view/jcsilva
Mobile: +55 21 98886-1235

**Comment by Er9y714** [comment]
I agree with @RIO4FDS, Sometimes Windows restart even after disabling auto updates because there is a limit of time postponing the updates etc. 

You may check Windows event manager logs. 

Not an answer to your question but if you have restart files, you can try continuing from the latest backup point.
